index.html

```diff
@@ -190,4 +190,4 @@
 		</div>
 	</article>
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
I read through the linked stackOveflow article and it seems like its better to be safe than sorry in this case, but I'm really glad to be rid of that icon at the end of my code! thanks